docs/requirements.txt

```diff
@@ -2,4 +2,4 @@ Sphinx==3.1.2
 furo
 recommonmark>=0.5.0
 sphinx-copybutton
-sphinxext-opengraph
```

Comment:
It looks like this line didn't actually change; can you checkout the old requirements.txt so that this file doesn't have a change here?